graph/mutators/comment.js

```diff
@@ -219,20 +220,62 @@ const addCommentTag = ({user, loaders: {Comments}}, {id, tag}) => {
 
 /**
  * Removes a tag from a Comment
- * @param {String} id          identifier of the comment  (uuid)
+ * @param {String} id      identifier of the comment  (uuid)
  * @param {String} tag     name of the tag
  */
 const removeCommentTag = ({user, loaders: {Comments}}, {id, tag}) => {
   return CommentsService.removeTag(id, tag);
 };
 
+/**
+ * Edit a Comment
+ * @param {String} id         identifier of the comment  (uuid)
+ * @param {Object} edit       describes how to edit the comment
+ * @param {String} edit.body  the new Comment body
+ */
+const editComment = async ({user, loaders: {Comments}}, {id, edit}) => {
+  const {body} = edit;
+  const comment = await CommentsService.findById(id);
```

Comment:
@wyattjoh I'm glad you can help me optimize this. Right now that findById is also needed to look up that comment's asset_id to apply the appropriate (e.g. bad word) filtering before deciding whether the update should even be made. Can your proposal accomodate this and still save a mongo query?